python/lsst/meas/algorithms/brightStarStamps.py

```diff
@@ -0,0 +1,295 @@
+# This file is part of meas_algorithms.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+"""Collection of small images (stamps), each centered on a bright star.
+"""
+
+__all__ = ["BrightStarStamp", "BrightStarStamps"]
```

Comment:
I know you will not see this in a lot of code we have, because people often get this wrong (reviewers included) but __all__ needs to be the first thing in the file, before the copyright

Reply:
No. https://developer.lsst.io/python/style.html?highlight=__all__#standard-code-order-should-be-followed 

To be specific:

* Shebang line, #! /usr/bin/env python (only for executable scripts)
* Module-level comments (such as the license statement)
* Module-level docstring
* `__all__ = [...]` statement, if present

Reply:
This is getting further off topic, but I notice that `__future__` imports aren't in that list in the dev guide, and I've been assuming they need to go before `__all__`.  Anyone know if that's correct?

Reply:
Jim if you are thinking about the annotation future import, it MUST be the absolute first thing in a file, because it changes how the interpreter works.

Reply:
I'm pretty sure python complains if the `__future__` is not the first line of code since the future applies to the entire file.

Reply:
Other future things may or may not be safe to be put other places.

Reply:
and by first, I mean before __all__ or any other executed code, things like docstring and comments are not interpreted and or parsed, so it does not really matter

Reply:
In case you care to know why, here is the page on future https://docs.python.org/3/library/__future__.html. basically there can be a CompilerFlag that changes how code is compiled into python byte code.